framework/bug-mining/README.md

```diff
@@ -36,164 +37,213 @@ following files:
 
 The **project id** should **start with an upper-case letter** and should be
 **short yet descriptive** (keep in mind that this id is used for commands such
-as `defects4j checkout -p <project_id>`). The **project name** should be
-**hyphenated and must not include spaces**.
-
-For example, the project name for the Apache Commons-Lang project, already
-included in Defects4J, is *commons-lang*, and its project id is *Lang*.
+as `defects4j checkout -p <project_id>`). The **project name** chould be
+**hyphenated but must not include spaces**. For example, the project name for
+the Apache Commons-Lang project, already included in Defects4J, is
+*commons-lang*, and its project id is *Lang*.
 
 2. Adapt the Perl module and the following properties if necessary:
     - Version control system (default is Git)
     - Location of the version control repository (default is
       `bug-mining/project_repos/<project_name>.git`)
     - Project layout (source and test directories)
 
+**TODO: What exactly may need to be adapted (i.e., which functions) and how
+should be adapted? Maybe we could add some TODOS to identify which bits may need
+to be checked/adapted, e.g., an example on how to adapt the perl module to SVN
+and how to configure a different source/test directory.**
+
 3. Adapt the Wrapper build file
 
+**TODO: Perhaps we could add examples of common issues, e.g., how to add project
+specific dependencies to the build file.**
+
 Identifying candidate bugs (populating the `commit-db`)
 -------------------------
 1. Bug mining starts with identifying the issue tracker for the project you are
    interested in. Defects4j's bug-mining framework supports:
-    - google (google-code)
-    - jira (defaulting to apache)
-    - github
-    - sourceforge (sf)
+    - [google-code](https://code.google.com/) ('google' for short)
+    - [jira](https://issues.apache.org/jira/)
```

Comment:
Is apache's jira the default or the only one supported right now?

Reply:
There is no need to define a default _tracker system_ as the `tracker-type` parameter of the `download-issues.pl` script isn't optional. And we currently support the following ones: google-code, jira, github, and sourceforge.